### ISSUE TITLE
Game crashes at startup

### ISSUE THREAD

**Issue by GengGode** [SpectrumQT/WWMI#3]
![image](https://github.com/SpectrumQT/WWMI/assets/47711102/5ca6098b-1e3b-4644-a0e5-321277526a36)
As mentioned in the title, when running Loader.exe to start the game, the pop-up window crashes. I am not sure what caused it

**Comment by SpectrumQT** [comment]
Hello! Please specify your GPU, Windows version and OS Language. Also, I need your logs to look further into the issue. Please follow the steps below:

1. Open **d3dx.ini** in **WWMI folder**, scroll to the end, and replace **[Logging]** section with:
  ```
  [Logging]
  calls=1
  input=0
  debug=1
  unbuffered=1
  force_cpu_affinity=1
  convergence=0
  separation=0
  debug_locks=1
  crash=1
  ```
2. Save changes to **d3dx.ini**
3. Start the game and wait until it crashes
4. Zip entire **WWMI folder**
5. Zip entire **\Wuthering Waves Game\Client\Saved\Logs folder**
6. Upload both archives to any cloud storage (i.e. Google Drive) and give me public link
7. Replace **d3dx.ini** again with one from WWMI download to disable logging

**Comment by SpectrumQT** [comment]
Reported issue is most likely fixed in [v0.6.2 update](https://github.com/SpectrumQT/WWMI/releases/tag/v0.6.2). Feel free to re-open it with logs provided if it'll still persist after update.
